collectors/collectors.go

```diff
@@ -18,8 +18,30 @@ package collectors
 
 import (
 	"time"
+
+	"github.com/prometheus/client_golang/prometheus"
 )
 
 var (
 	resyncPeriod = 5 * time.Minute
+
+	kubeStateMetricsSubsystem = "ksm"
+
+	ScrapeErrorTotalMetric = prometheus.NewCounterVec(
+		prometheus.CounterOpts{
+			Subsystem: kubeStateMetricsSubsystem,
```

Comment:
This is likely to disappear in the next version of the Prometheus client so I would prefer to just prepend the prefix everywhere.

Reply:
Yep. Just leanred this change from https://github.com/prometheus/client_golang/issues/240, will do.